meta-ibm/recipes-phosphor/webui/webui-vue/change-logo.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Start the change logo script
+
+[Service]
+Restart=no
+ExecStart=sh /usr/share/www/changeLogo.sh
```

Comment:
Lets just call it directly: "Execstart=/usr/share/www/changeLogo.sh"

Reply:
Done